topics/kentico-kontent/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Ilesh Mistry
+display_name: Kentico Kontent
+github_url: https://github.com/kentico
+logo: kontent.png
+related: cms, content-delivery
+released: 2016
+short_description: Kentico Kontent provides environment for easy content collaboration and features flexible API to power modern websites and apps.
+topic: kentico-kontent
+url: https://kontent.ai
+---
+Kentico Kontent is a Content as a Service system that replaces legacy CMS. It offers unified environment for content collaboration, flexible APIs for content management and delivery, content delivery network and SDKs for building websites and apps with the modern development stack.
```

Comment:
Thoughts on this?
```suggestion
Kentico Kontent is a Content as a Service system that can replace a legacy CMS. It offers a unified environment for content collaboration, flexible APIs for content management and delivery, and content delivery network and SDKs for building websites and apps with a modern development stack.
```

Reply:
lgtm

Reply:
@kenyonj I've made the changes and hopefully it's okay now?